### ISSUE TITLE
lb-ng fails for nestRemoting

### ISSUE THREAD

**Issue by clarkorz** [strongloop/loopback-sdk-angular#115]
```
var User = Model.extend('User');
var Product = Model.extend('Product');
var Review = Model.extend('Review');

User.hasMany(Product);
Product.hasMany(Review);

User.nestRemoting(Product);
```

> var targetClass = modelPrototype[scopeName]._targetClass;
>                                              ^
> TypeError: Cannot read property '_targetClass' of undefined
>     at buildScopeMethod (/usr/local/lib/node_modules/strongloop/node_modules/loopback-sdk-angular-cli/node_modules/loopback-sdk-angular/lib/services.js:107:46)

`Model.nestRemoting()` will create remote methods like `prototype.__findById__products__reviews`, but it doesn't create scope `products__reviews`.

**Comment by kennethlynne** [comment]
Same error I got with a simple remoteMethod

**Comment by kennethlynne** [comment]
Check out #160

**Comment by bajtos** [comment]
Closing as fixed by #168.
